API/ControllerAttributes/AllowedExtensionsAttribute.cs

```diff
@@ -0,0 +1,83 @@
+/*
+* Digital Excellence Copyright (C) 2020 Brend Smits
+* 
+* This program is free software: you can redistribute it and/or modify 
+* it under the terms of the GNU Lesser General Public License as published 
+* by the Free Software Foundation version 3 of the License.
+* 
+* This program is distributed in the hope that it will be useful, 
+* but WITHOUT ANY WARRANTY; without even the implied warranty 
+* of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. 
+* See the GNU Lesser General Public License for more details.
+* 
+* You can find a copy of the GNU Lesser General Public License 
+* along with this program, in the LICENSE.md file in the root project directory.
+* If not, see https://www.gnu.org/licenses/lgpl-3.0.txt
+*/
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace API.ControllerAttributes
+{
+    /// <summary>
+    /// Attribute for settings the allowed file extensions for every single file in the request.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method)]
+    public class AllowedExtensionsAttribute : ActionFilterAttribute
```

Comment:
Misschien kan dit beter verandert worden naar AllowedFileExtensions? Niet heel belangrijk maar maakt het denk ik wel wat duidelijker.

Reply:
Good suggestions not that it really maters to much but I will change it